spec/lib/open_food_network/lettuce_share_report_spec.rb

```diff
@@ -1,30 +1,32 @@
+require 'spec_helper'
+
 require 'open_food_network/lettuce_share_report'
 
 module OpenFoodNetwork
   xdescribe LettuceShareReport do
     let(:user) { create(:user) }
     let(:report) { LettuceShareReport.new user, {}, true }
-    let(:v) { create(:variant) }
+    let(:variant) { create(:variant) }
 
     describe "grower and method" do
       it "shows just the producer when there is no certification" do
         report.stub(:producer_name) { "Producer" }
         report.stub(:certification) { "" }
 
-        report.send(:grower_and_method, v).should == "Producer"
+        report.send(:grower_and_method, variant).should == "Producer"
       end
 
       it "shows producer and certification when a certification is present" do
         report.stub(:producer_name) { "Producer" }
         report.stub(:certification) { "Method" }
 
-        report.send(:grower_and_method, v).should == "Producer (Method)"
+        report.send(:grower_and_method, variant).should == "Producer (Method)"
       end
     end
 
     describe "gst" do
       it "handles tax category without rates" do
-        report.send(:gst, v).should == 0
+        report.send(:gst, variant).should == 0
```

Comment:
Since these methods are private, we should test the public methods that call them instead of them. Can you take a look if this is doable or create an issue otherwise @luisramos0 ?

Reply:
I see, I have removed send from the calls to table because it's not private.
For the other two gst and grower_and_method, the table method is returning their results directly in a data structure, so testing these methods individually is not too bad imo.
https://github.com/openfoodfoundation/openfoodnetwork/blob/8b1c1881f731d6b38290fea3c4ee7bcd69bc2c8b/lib/open_food_network/lettuce_share_report.rb#L33

we do have quite a bit of private methods testing in OFN. shall I create a generic issue with reference to this particular case?

Reply:
ping @sauloperez